scripts/results/plot_source_stats.py

```diff
@@ -26,30 +26,33 @@
     print("processing subject: %s" % subject)
     data_path = op.join(meg_dir, subject)
     contrast = mne.read_source_estimate(op.join(data_path, 'contrast-morphed'))
-    contrast.resample(100)
     contrast.crop(0., None)
+    contrast.bin(1. / 100.)  # simple way to get to 160 Hz sample rate
+    assert contrast.tstep == 0.01
     contrasts.append(contrast.data.T)
 
-X = np.abs(np.array(contrasts))
+X = np.array(contrasts)
+assert X.min() >= 0  # these already have the absolute value taken during inv
 
-connectivity = spatial_tris_connectivity(grade_to_tris(4))
+fsaverage_src = mne.read_source_spaces(op.join(subjects_dir, 'fsaverage',
+                                               'fsaverage-ico-5-src.fif'))
+connectivity = spatial_src_connectivity(fsaverage_src)
 p_threshold = 0.001
-t_threshold = -stats.distributions.t.ppf(p_threshold / 2., 15)
+t_threshold = -stats.distributions.t.ppf(p_threshold / 2., len(contrasts) - 1)
 
-stat_fun = partial(ttest_1samp_no_p, sigma=0.5)
+stat_fun = partial(ttest_1samp_no_p, sigma=0.01)
 T_obs, clusters, cluster_p_values, H0 = clu = \
     spatio_temporal_cluster_1samp_test(X, connectivity=connectivity, n_jobs=1,
                                        threshold=t_threshold,
                                        stat_fun=stat_fun, verbose=True)
 
-good_cluster_inds = np.where(cluster_p_values < p_threshold)[0]
+good_cluster_inds = np.where(cluster_p_values < 0.05)[0]
```

Comment:
we don't use `p_threshold` anymore?

Reply:
it is used to compute `t_threshold` which is used as the *initial* thresholding during clustering

Reply:
BTW this code is untested currently because I'm still doing `recon-all`s

Reply:
okay, before we merge this PR, could you share the html pages you built on your fork + a summary of the changes because they have to reflect in the manuscript (sort of like the what's new I guess :).